cli/README.md

```diff
@@ -0,0 +1,30 @@
+# Intro to the Command Line/Terminal
```

Comment:
I'm sorry trucnguyen. I'm still very confused on this reference. Are you asking to link the readme on index.html, or the index.html on readme? If so, how to reference it? Wish I could be further help.